src/components/Custom/CustomDonateCol/CustomDonateCol.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/fonts';
+@import '../../../styles/colors';
+
+.custom-donate-col {
```

Comment:
I feel this naming convention is pointless, why not just call it `.donate-col` ? Adding the custom in front doesn't add any thing to its utility